locksmith/src/utils/priceConversion.ts

```diff
@@ -28,7 +28,12 @@ export default class PriceConversion {
         `https://api.coinbase.com/v2/prices/${currency}-USD/buy`
       )
 
+      if (!response.ok) {
+        return undefined
+      }
+
       const { data } = await response.json()
+
       if (!data?.amount) {
         return 0
```

Comment:
Maybe we also handle things here... if there is no `amount` I think we can assume it's not "convertible"